frontend/src/components/login/login-input/LoginInput.tsx

```diff
@@ -138,6 +170,21 @@ const Login = () => {
           errorMessage={errorMsg}
         />
       )}
+      {/* This feature is experimental and should only be rendered on the demo URL (/login/sgid) */}
```

Comment:
`nit:` Do you mean `/ogp-login` instead of `/login/sgid`?

Reply:
Yup, should be `/ogp-login` instead. I've updated it.

Reply:
Thanks!